contracts/test/vault/upgrade.js

```diff
@@ -0,0 +1,30 @@
+const { expect } = require("chai");
+const { defaultFixture } = require("../_fixture");
+const { loadFixture } = require("../helpers");
+
+describe("VaultAdmin Upgrades", async function () {
+  let ousd, vault, vaultStorage, governor;
+
+  beforeEach(async function () {
+    const fixture = await loadFixture(defaultFixture);
+    vault = fixture.vault;
+    ousd = fixture.ousd;
+    governor = fixture.governor;
+    vaultStorage = await hre.ethers.getContractAt(
+      "VaultStorage",
+      vault.address
+    );
+  });
+
+  it("should upgrade to a new admin implementation", async function () {
+    const newVaultImpl = ousd.address; // ;)
```

Comment:
nit: clean up end empty comment

Reply:
Lol, I failed at humor.

The comment is supposed to represent  a winking face at the absurdity of setting the vault implementation address to the OUSD proxy address. 😄

Reply:
I'm fine waiting for this change to be rolled into future upgrades.

Reply:
I'm a code review robot. I do not understand humor from you, human // :)